pkg/storage/layers/connection.yaml

```diff
@@ -0,0 +1,76 @@
+definitions:
+  Connection:
+    description: Connection information
```

Comment:
Perhaps "details of connection types supported for accessing data stores. Not all are necessarily supported by fybrik storage allocation mechanism used in write and copy flows"

Reply:
Changed the description